tutorials/interactive-example/README.md

```diff
@@ -0,0 +1,8 @@
+# Interactive Example
```

Comment:
This is just my opinion so I'd love to know WDYT @yeya24 @metonymic-smokey

Reply:
I think `CONTRIBUTING.md` could have a link to this file?

Reply:
Yes, that sounds good. Just make sure to put the complete GitHub URL (like `https://github.com/thanos-io/thanos/blob/main/examples/interactive/README.md`) otherwise it would get 404'd on website.

Reply:
@onprem done in the latest commit!

Reply:
@onprem This link is causing the docs check to fail since the link will become valid only when the PR is merged to main. Is there something I can do to fix it?

Reply:
Ah, no worries. I can go ahead and merge this as the link will start working after the merge.